Design/Vehicle-Sizing/Pressure Drop Calculator.py

```diff
@@ -0,0 +1,127 @@
+import math
+from scipy.optimize import fsolve 
```

Comment:
Docstring and shebang

Reply:
If this module is going to be useful for more than just Marginal Stability, we should put it in it's own repository and make it it's own package.